packages/inventory/src/containers/ServicePointProfile/index.tsx

```diff
@@ -123,7 +123,7 @@ const ServicePointProfile = (props: ServicePointsProfileTypes) => {
   };
   const [structure] = useSelector((state) =>
     structuresSelector(state, { ...filters, searchQuery: spId })
-  );
+  ) as LocationUnit[];
```

Comment:
should this not be LocationUnit | undefined?, to what variable will this be typing?